### ISSUE TITLE
Tramstation HFR piping issue

### ISSUE THREAD

**Issue by JavaMasterhmm** [tgstation/tgstation#62744]
## Round ID:

[Round ID]: # No specific round ID.

## Reproduction:
The default piping on tramstation has a issue in the HFR room (atmospherics). The fuel input pipe is blocked by the atmos SM pipe (orange pipe) on layer 3.
![image](https://user-images.githubusercontent.com/55258493/141250331-e2a430ce-94fa-4f20-9c4b-aa2d1a581a79.png)

How to reproduce: 

Step 1: Load up the default tramstation map.

Step 2: go to the HFR room.

Step 3: Build the HFR.

Step 4: Try placing down a pipe for the fuel input

The pipe can't be placed because there is one there already under the floor tile

Step 5: cry about it

**Comment by D4C-420** [comment]
Few things. 1 you technically don’t HAVE to build the HFR there. You can shift it over to the left by a tile. 2 that’s the pipe for the turbine and you can always just move it around so it does not interfere with the HFR. 3 you can move the locations of the fuel,moderator and waste ports along with the interface on any side of the donut along with changing the direction of the coolant.  Like half of doing atmos is properly pathing your piping.

 All that given atmos for tram does kinda suck given the lack of 1. space (literal space. The rocks cut you off pretty fast. I honestly feels similar in size to kilo if not more cramped) 2 the weird way the default piping for the turbine is ( connected through the mix chamber and  maint, the pipe is orange rather than black unlike every other map)3. The lack of a standalone real pipe for n2. There is only a pipe which directly leads to the air mix and then from there connects to the yellow loop.Unlike o2 which has a pipe which leads to the air mix and a pipe which leads to the yellow loop.  
 
Tldr: you can just build around the pipe or move it but tram atmos still kinda sucks, feeling more cramped (to me) than other maps mainly due to space (outside) being limited due to the red rocks along with some odd quirks which makes it more confusing then needed.
